Assets/Scripts/Enemy/EnemyView.cs

```diff
@@ -32,6 +37,36 @@ private void DestroyEnemy()
             Destroy(gameObject);
         }
 
+        public void DestroyEnemyView()
+        {
+            Destroy(gameObject);
+        }
+
+        public void FollowTarget(Vector3 targetPos)
+        {
+            if (GameManager.Instance.currentState.gameStateType == GameStateType.Replay) return;
```

Comment:
this should be a IsReplayMode() function in GameManager

Reply:
I have getting the error here so added it, will be removing the code.